views/posts/index.hbs

```diff
@@ -25,28 +25,24 @@
     <div class="post-container card" id="{{this._id}}">
       <div class="actual-post">
         <div class="profile-picture">
-          <a href="/profile/{{this.user._id}}">
-            <img
-              src={{this.user.profilePicture}}
-              width="100"
-              height="100"
-              class="rounded-circle"
-            />
-          </a>
+          <img
+            src={{this.user.profilePicture}}
+            width="100"
+            height="100"
+            class="rounded-circle"
+          />
         </div>
         <div class="post-body">
           <div class="post-header">
-            <a href="/profile/{{this.user._id}}">
```

Comment:
same here?

Reply:
Yes I've re-added them in now, thanks for spotting it